---
timestamp: '2024-10-13T01:47:09.054952'
level: INFO
issuer: instance_solver
event_type: metadata
data:
    log_file: exp_logs/9 AStar-full-log length1-15/astar_length_10_rovers_pp01-err-rate-0-5.yaml
    instance_id: rovers/pp01-err-rate-0-5
    search_algorithm: astar
    plan_length: 10
    domain_class: rovers
    instance_name: pp01-err-rate-0-5
    ground_plan:
    - (calibrate rover0 camera0 objective1 waypoint3)
    - (take_image rover0 waypoint3 objective1 camera0 high_res)
    - (communicate_image_data rover0 general objective1 high_res waypoint3 waypoint0)
    - (sample_rock rover0 rover0store waypoint3)
    - (navigate rover0 waypoint3 waypoint1)
    - (navigate rover0 waypoint1 waypoint2)
    - (communicate_rock_data rover0 general waypoint3 waypoint2 waypoint0)
    - (drop rover0 rover0store)
    - (sample_soil rover0 rover0store waypoint2)
    - (communicate_soil_data rover0 general waypoint2 waypoint2 waypoint0)
    lifted_plan:
    - calibrate
    - take_image
    - communicate_image_data
    - sample_rock
    - navigate
    - navigate
    - communicate_rock_data
    - drop
    - sample_soil
    - communicate_soil_data


---
timestamp: '2024-10-13T01:47:09.062593'
level: INFO
issuer: instance_solver
event_type: ground_repair
data:
    repair_length: 5
    repair: 'drop,(full ?y),precPos,-1

        sample_soil,(empty ?s),precPos,-1

        take_image,(have_image ?r ?o ?m),effPos,1

        communicate_soil_data,(communicated_soil_data ?x),effPos,1

        navigate,(have_soil_analysis ?x ?z),effPos,1'


---
timestamp: '2024-10-13T01:47:09.326275'
level: ERROR
issuer: instance_solver
event_type: error
data: "An error occurred during A* search: Command '['python3', '/home/nader/projects/lifted-white-plan-domain-repair/relaxation_generator/generate-asp-model.py',\
    \ '--instance', 'heuristic_tools/problem-in.pddl', '--domain', 'heuristic_tools/domain-in.pddl',\
    \ '-t', 'heuristic_tools/datalog.theory', '-m', 'heuristic_tools/datalog.model',\
    \ '--lpopt-preprocessor', '--grounder', 'none', '--domain-print', 'heuristic_tools/domain-out.pddl',\
    \ '--problem-print', 'heuristic_tools/problem-out.pddl']' returned non-zero exit\
    \ status 1.. Stack trace: Traceback (most recent call last):\n  File \"/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py\"\
    , line 82, in solve_instance\n    path, goal_node = searcher.find_path(logger=logger,\
    \ log_interval=log_interval)\n                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\
    \  File \"/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/search_algorithms.py\"\
    , line 51, in find_path\n    for neighbor in current_node.get_neighbors():\n \
    \                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/node.py\"\
    , line 173, in get_neighbors\n    next_node = Node(\n                ^^^^^\n \
    \ File \"/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/node.py\"\
    , line 86, in __init__\n    self.h_cost = self.compute_h_cost() if self.h_cost_needed\
    \ and self.lifted_action_sequence else 0\n                  ^^^^^^^^^^^^^^^^^^^^^\n\
    \  File \"/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/node.py\"\
    , line 136, in compute_h_cost\n    h_cost = h.evaluate(self.original_domain, task,\
    \ [(l,) for l in self.lifted_action_sequence])\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\
    \  File \"/home/nader/projects/lifted-white-plan-domain-repair/heuristic_tools/heuristic.py\"\
    , line 346, in evaluate\n    val = self.get_val()\n          ^^^^^^^^^^^^^^\n\
    \  File \"/home/nader/projects/lifted-white-plan-domain-repair/heuristic_tools/heuristic.py\"\
    , line 299, in get_val\n    ground(**GROUND_CMD)\n  File \"/home/nader/projects/lifted-white-plan-domain-repair/relaxation_generator/shortcuts.py\"\
    , line 120, in ground\n    subprocess.check_call(command)\n  File \"/home/nader/miniconda3/envs/planning/lib/python3.12/subprocess.py\"\
    , line 413, in check_call\n    raise CalledProcessError(retcode, cmd)\nsubprocess.CalledProcessError:\
    \ Command '['python3', '/home/nader/projects/lifted-white-plan-domain-repair/relaxation_generator/generate-asp-model.py',\
    \ '--instance', 'heuristic_tools/problem-in.pddl', '--domain', 'heuristic_tools/domain-in.pddl',\
    \ '-t', 'heuristic_tools/datalog.theory', '-m', 'heuristic_tools/datalog.model',\
    \ '--lpopt-preprocessor', '--grounder', 'none', '--domain-print', 'heuristic_tools/domain-out.pddl',\
    \ '--problem-print', 'heuristic_tools/problem-out.pddl']' returned non-zero exit\
    \ status 1.\n"


---
timestamp: '2024-10-13T01:47:09.329082'
level: INFO
issuer: instance_solver
event_type: time_spent
data: 0.0 hours = 0.0 minutes = 0.28 seconds


---
timestamp: '2024-10-13T01:47:09.403779'
level: INFO
issuer: batch_solver
event_type: general
data: Subprocess finished with no exception.


